### ISSUE TITLE
Core IoT Service - only valid for 1 month?

### ISSUE THREAD

**Issue by JeroenTuinstra** [tuya/tuya-home-assistant#791]
Does everyone notice that the Core IoT service of Tuya (trial) is only valid for 1 month?
Does that mean we have to renew every month to get this integration working?
Do we actually need Core IoT service?

**Comment by gdgeist** [comment]
How did you resolve - mine just expired?

**Comment by moyuehan** [comment]
My IoT Core had an expiration date on 2022-07-21 and all services over the Tuya cloud stopped working : e.g. Smart Plug Power.
In the Smart Life app on my cell phone all cloud functions were still working. The on/off - local functions - in HA were working. As I am using the "power" information in my HA automation, a immediate solution was needed. Lucky me, 3 days ago I applied for a "free basic resource pack" for IoT Core. This is now alive, expiration date 2022-08-21. I checked via "tech support" with a Tuya technician : free resource pack for IoT Core only lasts for 1 month. This is completely in contraction with Mark Watt Tech in his you tube video : there is NO AUTOMATIC renewal after the trial period expires. 
Previously I was able to get an extension for 6 months on the Tuya IoT Platform. This option I cannot find anymore. Gone.
"Subscribe to Resource Pack" cannot be done before the expiration date. I tried several times, and now I am not sure as of how many hours before or after expiration the "subscribe to resource pack" was possible. Looking at the IoT Platform screen for "subscribed resources" for IoT Core I can see my previous pack expired 2022-07-21 16:57:42 and my new pack came alive 2022-07-21 22:36:20. That is about a 6 hour gap the automation is not working.
